--- a/build/cmake/aom_optimization.cmake	2025-09-06 05:10:37
+++ b/build/cmake/aom_optimization.cmake	2026-02-01 21:48:02
@@ -215,11 +215,6 @@
   execute_process(COMMAND ${CMAKE_ASM_NASM_COMPILER} -hf
                   OUTPUT_VARIABLE nasm_helptext)
 
-  if(NOT "${nasm_helptext}" MATCHES "-Ox")
-    message(
-      FATAL_ERROR "Unsupported nasm: multipass optimization not supported.")
-  endif()
-
   if("${AOM_TARGET_CPU}" STREQUAL "x86")
     if("${AOM_TARGET_SYSTEM}" STREQUAL "Darwin")
       if(NOT "${nasm_helptext}" MATCHES "macho32")
