diff --git a/CMakeLists.txt b/CMakeLists.txt
index ed4948f0..8be3402a 100644
--- a/CMakeLists.txt	2025-08-27 22:57:53
+++ b/CMakeLists.txt	2025-08-27 22:59:04
@@ -254,3 +254,6 @@
 # create the .deb and .rpm packages using cpack
 include ( CM_linux_packages.cmake )
 
+# generate pkgconfig file:
+install ( CODE "execute_process(COMMAND sh ${PROJECT_SOURCE_DIR}/gen_pc.sh)")
+
Index: gen_pc.sh
===================================================================
--- a/gen_pc.sh	(nonexistent)
+++ b/gen_pc.sh	(working copy)
@@ -0,0 +1,10 @@
+cat > ${JHBUILD_PREFIX}/lib/pkgconfig/libyuv.pc <<EOF
+prefix=${JHBUILD_PREFIX}
+libdir=${JHBUILD_PREFIX}/lib
+includedir=${JHBUILD_PREFIX}/include
+
+Name: libyuv
+Description: YUV conversion and scaling functionality library
+Version: 0
+Libs: -L${libdir} -lyuv
+EOF
