--- a/Makefile.in
+++ b/Makefile.in
@@ -151 +151 @@
-	ctags-recursive dvi-recursive html-recursive info-recursive \
+	ctags-recursive dvi-recursive info-recursive \
@@ -153 +153 @@
-	install-exec-recursive install-html-recursive \
+	install-exec-recursive \
@@ -407 +406,0 @@
-htmldir = @htmldir@
@@ -921,4 +919,0 @@
-html: html-recursive
-
-html-am:
-
@@ -937,4 +931,0 @@
-install-html: install-html-recursive
-
-install-html-am:
-
@@ -986 +977 @@
-	dvi-am html html-am info info-am install install-am \
+	dvi-am info info-am install install-am \
@@ -988,2 +979,2 @@
-	install-dvi-am install-exec install-exec-am install-html \
-	install-html-am install-info install-info-am install-man \
+	install-dvi-am install-exec install-exec-am install-info \
+	install-info-am install-man \
